packages/opentelemetry-node/test/NodeTracerProvider.test.ts

```diff
@@ -56,7 +56,7 @@ describe('NodeTracerProvider', () => {
   afterEach(() => {
     // clear require cache
     Object.keys(require.cache).forEach(key => delete require.cache[key]);
-    provider.stop();
+    // provider.stop(); TODO: call stop on the enabler
```

Comment:
Looks like this is only working with a provider... is this `TODO` still accurate? Seems like it is referring to what became the `NodePluginManager` that is not in play here yet.

Reply:
it's been moved to Plugin Manager tests